src/utils/file/index.ts

```diff
@@ -578,3 +579,20 @@ export function getLatestVersionFiles(files: EnteFile[]) {
         (file) => !file.isDeleted
     );
 }
+
+export function getUserPersonalFiles(files: EnteFile[]) {
```

Comment:
can this logic be part of `getTotalFileCount`? I don't see us reusing it anywhere else as of now. 

```ts
export const getUserPersonalFileCount = async (): Promise<number> => {
    try {
        const user: User = getData(LS_KEYS.USER);
        if (!user?.id) {
            throw Error('user missing');
        }
        const files: EnteFile[] = await getLocalFiles();
        const userPersonalFiles = files.filter((file) => file.ownerID === user.id);
        return userPersonalFiles.length;
    } catch (e) {
        logError(e, 'getUserPersonalFileCount failed');
        throw e;
    }
};
```